content/blog/2018/istio-twitch-stream/index.md

```diff
@@ -21,6 +21,30 @@ The stream is going to be a full day of Istio content. Hopefully we'll have a go
 
 It's easy! Just navigate [here](https://twitch.tv/ibmcode) on August 17th starting at 10 AM Pacific.
 
+## Schedule
+
+All times are `PDT`.
+
+| Time | Speaker | Affiliation |
+| --- | --- | --- |
+| 10:00 - 10:30 | `Spencer + Lisa` | `IBM / Portworx` |
```

Comment:
i'd recommend same syntax throughout this change. e.g. `Spencer Krum / Lisa (don't know last name:)